src/net.cpp

```diff
@@ -667,6 +676,19 @@ bool CNode::ReceiveMsgBytes(const char *pch, unsigned int nBytes)
         nBytes -= handled;
 
         if (msg.complete()) {
+
+            //store received bytes per command
+            //to prevent a memory DOS, only allow valid commands
+            std::string command(msg.hdr.pchCommand);
+            for (unsigned int i = 0; i < sizeof(logAllowIncommingCmds)/sizeof(logAllowIncommingCmds[0]); i++)
```

Comment:
Yes. Would be possible. I somehow searched for a solution that doesn't require initialization. Do you think the `map.find()` would be faster or just less code?